### DISCUSSION TITLE
Welcome to UCF-SST-CitySim-Dataset Discussions!

### DISCUSSION THREAD

**Discussion by ozheng1993** [ozheng1993/UCF-SST-CitySim-Dataset#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Taipingchangan** [comment]
Hello, I am a student from China. After reading this data set, I still have some doubts about the signal control at the intersection of University@Alafaya. Could you please tell me more about the signal phase at the intersection of University@Alafaya?

**Reply by ozheng1993** [reply]
What is your question?

**Reply by Taipingchangan** [reply]
Thanks for your reply. I'm wondering what's the signal phase at the intersection of University@Alafaya(in CitySim Dataset). Could you please tell me the signal phase at the intersection of University@Alafaya? I'll be appreciate it if you can tell me more about it.

**Reply by ozheng1993** [reply]
In the Final Release file, you can find a signal timing file. And signal timing documentation here: https://github.com/ozheng1993/UCF-SST-CitySim-Dataset/wiki

**Reply by Taipingchangan** [reply]
Thanks for your repy! Thank you!

**Comment by Shenjiayan99** [comment]
Hello, I am a  student, I have a question for this dataset, could you please tell me if the FreewayC dataset contains congested flow? I'll be appreciate it if you can tell me  about it.

**Reply by ozheng1993** [reply]
should have

**Comment by Taipingchangan** [comment]
您好，已收到你的来信，如果后续还有数据需求，可以加入还数据集开源人员组织的微信群聊，如有需要我可以向群主申请邀请你进群。最后顺便问一下，您是怎么找到我的邮箱的？我不清楚我的邮箱及相关信息是怎么泄露的。谢谢！



---Original---
From: ***@***.***&gt;
Date: Mon, Apr 29, 2024 10:26 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [UCF-SST-Lab/UCF-SST-CitySim1-Dataset] Welcome toUCF-SST-CitySim-Dataset Discussions! (Discussion #1)




 
您好，我是一名学生，这个数据集现在还可以下载吗，目前的申请表页无法打开，请问是否可以通过其他方式获取申请表呢 ***@***.***，如果您能帮助我，我将不胜感激。

**Reply by shanghuoliangcha123** [reply]
您好 我是一名学生  刚才GitHub申请表页面无法进入 想在GitHub官网下面评论获取其他获得申请表的渠道  故在您能的评论下评论 并不是您的邮箱 如果可以的话 您是否可以将我邀请入群

**Comment by Taipingchangan** [comment]
可以，请留下您的微信联系方式以及姓名备注等信息，谢谢！



---Original---
From: ***@***.***&gt;
Date: Mon, Apr 29, 2024 10:56 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [UCF-SST-Lab/UCF-SST-CitySim1-Dataset] Welcome toUCF-SST-CitySim-Dataset Discussions! (Discussion #1)




 
您好 我是一名学生  刚才GitHub申请表页面无法进入 想在GitHub官网下面评论获取其他获得申请表的渠道  故在您能的评论下评论 并不是您的邮箱 如果可以的话 您是否可以将我邀请入群

**Comment by shanghuoliangcha123** [comment]
微信17832906197 姓名石帅坤  非常感谢！



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年04月29日 10:58 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [UCF-SST-Lab/UCF-SST-CitySim1-Dataset] Welcome to UCF-SST-CitySim-Dataset Discussions! (Discussion #1) |

可以，请留下您的微信联系方式以及姓名备注等信息，谢谢！



---Original---
From: ***@***.***&gt;
Date: Mon, Apr 29, 2024 10:56 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [UCF-SST-Lab/UCF-SST-CitySim1-Dataset] Welcome toUCF-SST-CitySim-Dataset Discussions! (Discussion #1)





您好 我是一名学生 刚才GitHub申请表页面无法进入 想在GitHub官网下面评论获取其他获得申请表的渠道 故在您能的评论下评论 并不是您的邮箱 如果可以的话 您是否可以将我邀请入群
